[doc] fix amp doc

guangyey opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #125673

# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



cc @mcarilli @ptrblck @leslie-fang-intel @jgong5

guangyey commented:
@albanD Sorry, I made a mistake on the document `is_autocast_available`. This PR aims to fix it.

guangyey commented on docs/source/conf.py:
```diff
@@ -159,6 +159,7 @@
     "hierarchical_pickle",
     # torch.amp.autocast_mode
     "autocast_decorator",
+    "is_autocast_available",
```
OK, let me check.

guangyey committed:
fix amp doc

[ghstack-poisoned]

guangyey committed:
Update on "fix amp doc"




[ghstack-poisoned]

guangyey committed:
Update on "fix amp doc"




[ghstack-poisoned]

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



[ghstack-poisoned]

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



[ghstack-poisoned]

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



[ghstack-poisoned]

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



cc mcarilli ptrblck leslie-fang-intel jgong5

[ghstack-poisoned]

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



cc mcarilli ptrblck leslie-fang-intel jgong5

[ghstack-poisoned]

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



cc mcarilli ptrblck leslie-fang-intel jgong5

[ghstack-poisoned]

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



cc mcarilli ptrblck leslie-fang-intel jgong5

[ghstack-poisoned]

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



cc mcarilli ptrblck leslie-fang-intel jgong5

[ghstack-poisoned]

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



cc mcarilli ptrblck leslie-fang-intel jgong5

[ghstack-poisoned]

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



cc mcarilli ptrblck leslie-fang-intel jgong5

[ghstack-poisoned]

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



cc mcarilli ptrblck leslie-fang-intel jgong5

[ghstack-poisoned]

albanD commented:
CI seems to be not so happy right now?

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



cc mcarilli ptrblck leslie-fang-intel jgong5

[ghstack-poisoned]

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



cc mcarilli ptrblck leslie-fang-intel jgong5

[ghstack-poisoned]

guangyey commented:
> CI seems to be not so happy right now?

I can't fully understand the syntax of these docs, so I'm experimenting with them over and over again :(

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



cc mcarilli ptrblck leslie-fang-intel jgong5

[ghstack-poisoned]

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



cc mcarilli ptrblck leslie-fang-intel jgong5

[ghstack-poisoned]

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



cc mcarilli ptrblck leslie-fang-intel jgong5

[ghstack-poisoned]

albanD reviewed (commented):
Yes they are a bit finicky. Especially for re-exposed APIs.

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



cc mcarilli ptrblck leslie-fang-intel jgong5

[ghstack-poisoned]

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



cc mcarilli ptrblck leslie-fang-intel jgong5

[ghstack-poisoned]

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



cc mcarilli ptrblck leslie-fang-intel jgong5

[ghstack-poisoned]

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



cc mcarilli ptrblck leslie-fang-intel jgong5

[ghstack-poisoned]

guangyey committed:
Update on "[doc] fix amp doc"


# Motivation
The doc shown on https://pytorch.org/docs/main/amp.html#torch.amp.autocast_mode.is_autocast_available should be `torch.amp.is_autocast_available` instead of `torch.amp.autocast_mode.is_autocast_available`. This PR aims to fix it.



cc mcarilli ptrblck leslie-fang-intel jgong5

[ghstack-poisoned]

guangyey commented:
Close this PR as it is expected.
